go.mod

```diff
@@ -95,6 +95,9 @@ require (
 // work.
 replace github.com/golang/glog => ./build/replacements/github.com/golang/glog
 
-// Before 0.15.0 OPA was not using go modules. We must retract these releases in order to have pkg.go.dev
-// show the latest versions. See: https://github.com/golang/go/issues/58925
+// retract directive comment below will be displayed as a warning on pkg.go.dev for the old package name. Please retain
+// this for future releases.
+
+// Use the path github.com/open-policy-agent/opa (lower-case), not ...open-policy-agent/OPA.
```

Comment:
```suggestion
// Use the path github.com/open-policy-agent/opa (lower-case), not github.com/open-policy-agent/OPA.
```

Reply:
I considered this, but didn't want page rank to find any more references to the old URL than strictly required... perhaps that just makes it confusing.

Reply:
Ah OK.... I guess I was in the "confused" camp 😅

Reply:
Would it show up as a link even?

Reply:
It looks like not even the https:// part becomes link 
<img width="1365" alt="image" src="https://user-images.githubusercontent.com/870638/226572650-b264cba1-0fc0-46b6-a5e6-a111dd02ffed.png">

Reply:
Either way, with `...` or with `github.com/` is fine with me. Merge whatever you prefer 😄

Reply:
After some digging, it doesn't appear so: https://pkg.go.dev/github.com/Shopify/sarama@v1.32.0

Reply:
I have updated it now 🙂